superduperdb/base/datalayer.py

```diff
@@ -373,6 +347,23 @@ def predict(
             return Document(out), [Document(x) for x in context]
         return Document(out), []
 
+    def validate(self, query: ExecuteQuery, *args, **kwargs) -> None:
+        """
+        Validate a query on the db.
+
+        :param query: select, insert, delete, update,
+        """
+        if not isinstance(query, (Delete, Select, Update)):
+            return
+        assert isinstance(query, (Delete, Select, Update))
```

Comment:
I believe an assertion here was added for debug purposes and should be removed from the final version.
```suggestion
        if not isinstance(query, (Delete, Select, Update)):
            return
```

Reply:
yup,
I had few leftovers from my debugging activities on this function, removed them 
Thanks